### ISSUE TITLE
Some observations with next branch.

### ISSUE THREAD

**Issue by yyr** [magit/magit#1529]
I am trying out next branch, I have couple of problems with it. 

The major one. There is no branch manager any more. i.e, I can't see  `b v` to invoke branch manager. am I missing anything to load?

There is no `C` option to take directly to commit-mode with log entered for current file. 

TAB reveal changed the order. In status buffer, a TAB press on file item used to reveal everything, now it shows the chunks folded instead. I need press TAB another time to reveal every thing of the file item.

in other words I like to have.
`TAB`  -> reveal every thing
`TAB`  ->  show folded chunks.
`TAB`  ->  fold everything back to end the cycle.

In fact presanally I don't even need second loop in the cycle since I commit often so I always have small diffs.

Thank you for the on magit.
